### PR TITLE
Nerf Space Loot 

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do

- Remove guarantee elite modsuit from syndicate KM in depot. Instead, have a chance to have one in locker.
- Replace all guarantee blood-red MODsuits in ruins with pre-builded miner modsuit. Now you can obtain them only after defeating Commando who wearing this, who are very few.
- Energy Cutlass now have new decription, lose his syndicate tech and lose PvP bonuses as armor penetration and embed bonuses

## Why It's Good For The Game

Too many good loot in space. Point is, instead of strict removing dangerous loot, we can make their obtaining much harder and their power is not as full as original syndicate  items. 
- QM boss is as easy to kill as normal crook, while guarding shitton of uplink loot and guaranteed elite modsuit. Too much for him.
- Cutlass is a copy of 40 TC esword, having only 20 damage instead of 30, which means you still have excellent armor penetration, good syndi techs and incredibly danger throwing impact. Too much for a most common weapon in space
- Blood-Red Modsuits have nearly the best armor you can possibly get, and having them more than 1-2 in space is also not good for any means.

## Images of changes
Syndi-Cake factory still has 1 BRM, but not 2. Miner suit lays in locker inside this room
image

Safe in one-half also contain miner modsuit, who has a lot of good modules for explorer
image

Energy cutlass now dont have an easiest access for 3 illegal techs, right as PvP bonuses
image

## Testing

Load maps, having corgi slaps, finding exact changed loot

<hr>

### Declaration

I have an Pre-approval!
image

## Changelog

:cl:
add: Add a chance to have Elite Modsuit in Depot armory lockers.
del: Depot QM now doesnt have Elite Modsuit after death.
del: Energy Cutlass now doesnt have additional Armor Penetration, Throwing bonuses and illegal techs.
tweak: Replace Blood-Red Modsuits in SyndyCake Factory locker and in Onehalf safe with loaded miner modsuits.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/melee/energy_melee_weapons.dm [hunk 1]
```diff
@@ -345,12 +345,17 @@
 //////////////////////////////
 /obj/item/melee/energy/sword/pirate
 	name = "energy cutlass"
-	desc = "Arrrr matey."
+	desc = "The Syndicate used to selling defective energy swords to suspicious persons, expecting nothing. But the technology was repaired with duct tape, and now this is the main weapon of space pirates. Arr, matey!"
```
[the remark this answers is not in the document]
Comment (Adrer) [reply code/game/objects/items/weapons/melee/energy_melee_weapons.dm:348]: Should be "its", not "it's"

File: code/game/objects/items/weapons/melee/energy_melee_weapons.dm [hunk 2]
```diff
@@ -345,12 +345,17 @@
...
 	force_on = 20
+	throwforce_on = 5 // No PvP shenanigans, this is main weapon in PvE explorer gameplay and can be obtained very easy
+	embed_chance = 45
+	embedded_impact_pain_multiplier = 4
+	armour_penetration_percentage = 0
+	armour_penetration_flat = 0
```
[the remark this answers is not in the document]
Comment (ErdGinalD) [reply code/game/objects/items/weapons/melee/energy_melee_weapons.dm:354]: Simple mobs do not have any armor, only pure HP.

File: _maps/map_files/RandomRuins/SpaceRuins/syndiecakesfactory.dmm [hunk 3]
```diff
@@ -628,7 +628,7 @@
 	},
 /area/ruin/space/syndicakefactory)
 "wW" = (
-/obj/machinery/suit_storage_unit/syndicate,
+/obj/machinery/suit_storage_unit/mining,
```
Comment (1080pCat) [inline _maps/map_files/RandomRuins/SpaceRuins/syndiecakesfactory.dmm:631]: is this intended to be a normal mining suit? the base version isn't space proof, only the asteroid skinned one is.

Comment (ErdGinalD) [reply _maps/map_files/RandomRuins/SpaceRuins/syndiecakesfactory.dmm:631]: inside of this storage is asteroid one
image

**Comment by ErdGinalD** [comment]
> Oooo boy thats a lot of changes.
> 
> * I don't think the cutlass should lose its illegals 2 (which gets you to level 3), illegals 3 main thing it unlocks is expanded storage, other then that, there isn't anything major or super important it unlocks and outside of an RnG dart pistol in maints, not many other ways to get illegals 3 without some kind of antag intervention.
> * The Nerfs of the cutlass are all good. _I believe it shouldn't be S-class anymore with these nerfs, it should be moved back down to C-class_. (I spoke to Qwerty about it, but from the point of view of officers being able to use them isn't a good thing, it is a bummer if you were being perma'd for a kinda eh weapon, but I guess that is why parole exists)
> 
> Other then those minor concerns, all this looks fine to me. What modules will be inside the mining modsuit, as that isn't stated on the PR?

Mining modsuit will be pre-builded, which contain this list of modules
image

**Comment by Stoniest** [comment]
> > Oooo boy thats a lot of changes.
> > 
> > * I don't think the cutlass should lose its illegals 2 (which gets you to level 3), illegals 3 main thing it unlocks is expanded storage, other then that, there isn't anything major or super important it unlocks and outside of an RnG dart pistol in maints, not many other ways to get illegals 3 without some kind of antag intervention.
> > * The Nerfs of the cutlass are all good. _I believe it shouldn't be S-class anymore with these nerfs, it should be moved back down to C-class_. (I spoke to Qwerty about it, but from the point of view of officers being able to use them isn't a good thing, it is a bummer if you were being perma'd for a kinda eh weapon, but I guess that is why parole exists)
> > 
> > Other then those minor concerns, all this looks fine to me. What modules will be inside the mining modsuit, as that isn't stated on the PR?
> 
> Mining modsuit will be pre-builded, which contain this list of modules image

Yep, all good. However, I'd still like you not to remove illegals tech from the cutlass.

**Review by Burzah — APPROVED** [review]

**Comment by Komrad822** [comment]
> * The Nerfs of the cutlass are all good. _I believe it shouldn't be S-class anymore with these nerfs, it should be moved back down to C-class_. (I spoke to Qwerty about it, but from the point of view of officers being able to use them isn't a good thing, it is a bummer if you were being perma'd for a kinda eh weapon, but I guess that is why parole exists)

Remember, Blood red magboots, which are JUST magboots painted red, are S-class still. Red=evil.

**Comment by Stoniest** [comment]
> > * The Nerfs of the cutlass are all good. _I believe it shouldn't be S-class anymore with these nerfs, it should be moved back down to C-class_. (I spoke to Qwerty about it, but from the point of view of officers being able to use them isn't a good thing, it is a bummer if you were being perma'd for a kinda eh weapon, but I guess that is why parole exists)
> 
> Remember, Blood red magboots, which are JUST magboots painted red, are S-class still. Red=evil.

Yea, I spoke to Qwerty about it, it's fine, I'd just like the illegals tech to stay due to there not being much that can give level 3 tech apart from completely random spawns in maints or an agent wasting TC.

**Comment by ErdGinalD** [comment]
there is no conflict now, choo choo

File: code/game/objects/effects/spawners/depot_spawners.dm [hunk 4]
```diff
@@ -167,6 +167,7 @@
 		/obj/item/shield/energy,
 		/obj/item/storage/box/syndie_kit/teleporter,
 		/obj/item/weaponcrafting/gunkit/universal_gun_kit,
+		/obj/item/mod/control/pre_equipped/traitor_elite = 1,
```
Comment (DGamerL) [inline code/game/objects/effects/spawners/depot_spawners.dm:170]:
```suggestion
		/obj/item/mod/control/pre_equipped/traitor_elite
```

**Review by DGamerL — COMMENTED** [review]
LGTM, the throwforce of the cutlass being on only being 5 seems maybe a bit too small for me, up it to 10 (this is a non-balance opinion)

**Review by DGamerL — APPROVED** [review]

**Comment by thraes** [comment]
Hello! This is my first post on github and I'm not really sure how this is supposed to work but as somebody who has played explorer for around 500 hours I would like to provide some input on this change. Replacing the syndicate modsuit with a mining modsuit is ok, but the one it has been replaced with appears to be unintentionally not spaceproof. It has the asteroid skin which is supposed to make mining modsuits space proof but I take constant burn damage wearing it in space. It appears something has been missed. 

I would also like to add that the modules on the modsuit are not great for exploring and that the gps module provides a marker of where the space ruins are, which I'm not sure is intentional. The complexity of the modsuit is totally full with all these mods and you need to remove them to make space for normal ion jetpack and welding mod from standard explorer suit. This is a modsuit being found at a syndicate facility. I think it would be fair to give them syndicate storage and/or advanced jetpack instead of ore bag, drill, gps, which are all immediately stripped off to make room for actually useful modules

**Comment by thraes** [comment]
[ErdGinalD] on their way to nerf explorers : roadrunner.mp4
on their way to fix the nerf: filenotfound.jpg

after playing, i must say this nerf has not really made it any harder for a single explorer to get a red modsuit. if anything, it is easier now because syndicake has a gps marker on it. what it HAS done is made it a lot harder to get an upgraded modsuit at all if there are four explorers and somebody beats you to one of the red suits, leaving you essentially sol because the mining mods are broken

**Pull request merged**